.github/workflows/build-test-lint.yml

```diff
@@ -5,13 +5,13 @@ jobs:
     name: build
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
+        go-version: [1.17.x, 1.18.x]
```

Comment:
Do we still need to support 1.17.x?

Reply:
I think it's in the build verification (we only publish 1.18 built container images)